test/unit/swap/UniswapV3SwapAdapter.t.sol

```diff
@@ -51,17 +54,17 @@ contract UniswapV3SwapAdapterUnitTests is Test {
 
     function test_constructor_zeroWrappedMToken() external {
         vm.expectRevert(IUniswapV3SwapAdapter.ZeroWrappedMToken.selector);
-        new UniswapV3SwapAdapter(address(0), swapFacility, UNISWAP_V3_ROUTER, admin, whitelistedToken);
+        new UniswapV3SwapAdapter(address(0), swapFacility, UNISWAP_V3_ROUTER, admin, new address[](0));
```

Comment:
Use the `whitelistedToken` variable instead.

```suggestion
        new UniswapV3SwapAdapter(address(0), swapFacility, UNISWAP_V3_ROUTER, admin, whitelistedToken);
```

Reply:
why address(0) was added ?